doc/api/n-api.md

```diff
@@ -241,15 +241,16 @@ from version 3 with some additions. This means that it is not necessary
 to recompile for new versions of Node.js which are
 listed as supporting a later version.
 
-|       | 1       | 2        | 3        | 4        | 5         |
-|-------|---------|----------|----------|----------|-----------|
-| v6.x  |         |          | v6.14.2* |          |           |
-| v8.x  | v8.0.0* | v8.10.0* | v8.11.2  | v8.16.0  |           |
-| v9.x  | v9.0.0* | v9.3.0*  | v9.11.0* |          |           |
-| v10.x | v10.0.0 | v10.0.0  | v10.0.0  | v10.16.0 | v10.17.0  |
-| v11.x | v11.0.0 | v11.0.0  | v11.0.0  | v11.8.0  |           |
-| v12.x | v12.0.0 | v12.0.0  | v12.0.0  | v12.0.0  | v12.11.0  |
-| v13.x | v13.0.0 | v13.0.0  | v13.0.0  | v13.0.0  | v13.0.0   |
+|       | 1       | 2        | 3        | 4        | 5         | 6         |
+|-------|---------|----------|----------|----------|-----------|-----------|
+| v6.x  |         |          | v6.14.2* |          |           |           |
+| v8.x  | v8.0.0* | v8.10.0* | v8.11.2  | v8.16.0  |           |           |
+| v9.x  | v9.0.0* | v9.3.0*  | v9.11.0* |          |           |           |
+| v10.x | v10.0.0 | v10.0.0  | v10.0.0  | v10.16.0 | v10.17.0  | v10.20.0  |
+| v11.x | v11.0.0 | v11.0.0  | v11.0.0  | v11.8.0  |           |           |
+| v12.x | v12.0.0 | v12.0.0  | v12.0.0  | v12.0.0  | v12.11.0  |           |
+| v13.x | v13.0.0 | v13.0.0  | v13.0.0  | v13.0.0  | v13.0.0   |           |
+| v14.x | v14.0.0 | v14.0.0  | v14.0.0  | v14.0.0  | v14.0.0   | v14.0.0   |
```

Comment:
The usual process is to add `REPLACEME` placeholders that get replaced in the release commit (i.e. `14.0.0`).

Reply:
@richardlau will that do the right thing when it gets backported?

Reply:
I think that depends on whether the commit is picked from current or master.

Reply:
Since this is a matrix adding REPLACEME for all versions will not do what we want. Maybe we should add REPLACEME for the version corresponding to the branch against which the commit is being applied, and hardcoded version values for all the other branches.

This means that when we backport this commit we'll necessarily have to edit it.

Reply:
I'm happy to break from convention/the usual process if it makes sense and does the right thing. I make a point of pointing out when we do diverge from the norm though so that we do so consciously.